_includes/demos/balance-php.php

```diff
@@ -1,33 +1,33 @@
 &lt;?php
```

Comment:
Looks like something is wrong with the `<` and `>` signs. Is it intentional?

Reply:
This approach was used for preventing from reading them as tags in code representation.